BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java

```diff
@@ -13,6 +13,7 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
+
```

Comment:
Again, please check your staged files and changes before committing. This is easily avoidable. Please revert